modules/data-virtualization/pom.xml

```diff
@@ -63,13 +63,6 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>3.0.0-M1</version>
-				<dependencies>
```

Comment:
The SDK generator generates unit tests that use TestNG, not JUnit, so we need to just remove the junit reference here.